src/Microsoft.OpenApi.Readers/ParseNodes/PropertyNode.cs

```diff
@@ -30,8 +30,7 @@ internal class PropertyNode : ParseNode
             IDictionary<string, Action<T, ParseNode>> fixedFields,
             IDictionary<Func<string, bool>, Action<T, string, ParseNode>> patternFields)
         {
-            Action<T, ParseNode> fixedFieldMap;
-            var found = fixedFields.TryGetValue(Name, out fixedFieldMap);
+            var found = fixedFields.TryGetValue(Name, out var fixedFieldMap);
```

Comment:
found is being assigned but never read, maybe this statement could be collapsed into the if condition